### ISSUE TITLE
Accidentally clicking on a different room while sharing a media doesn't allow to choose a different room

### ISSUE THREAD

**Issue by alittlebitofit** [element-hq/element-android#6210]
### Steps to reproduce

1. Select any media file to share on Element.
2. Element displays the list of rooms. Select any room.

#### What did you expect?
I expect an option to choose a different room if I accidentally select a wrong room.

#### What happened instead?
Pressing back exits the media and opens the wrongly selected room.

### Operating system version

Android 12

### Application version and app store

1.4.14 [40104140] (F-06830853)

### Homeserver

1.4.14 (06830853)

### Will you send logs?

No

**Comment by bmarty** [comment]
I agree the UX could be improved here. Note that you can long click on a room, to be able to select multiple rooms - and also deselect a room.
